src/tests/ftest/mpiio/romio.yaml

```
@@ -8,10 +8,24 @@ timeout: 150
 # romio test suite directory in CI nodes when available.
 server_config:
     name: daos_server
+    servers:
+        bdev_class: nvme
+        bdev_list: ["0000:81:00.0","0000:da:00.0"]
```

We could use placeholders here:
```suggestion
        bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
```